### PR TITLE
[v13] Edit forScopes configurations and edit guides

### LINKED ISSUE #26500: Add scoped components for Teleport Team
## Applies To

Here are all of the scoped components with a scope for `cloud`:

```bash
$ grep -REn 'scope=\{?\[?"cloud' docs/pages | wc -l
     227
```

Assuming it takes 5 minutes to create a `team`-scoped component for each `cloud`-scoped component, that's around 19 hours of work.

## Details
With Teleport Team coming soon/already here (depending on when you read this), we'll want to make sure our scoped components include details about Teleport Team. These sometimes overlap with our current `cloud`-scoped components, but not always.

First, we need to make `team` a valid scope in `gravitational/docs`.

### PR DESCRIPTION
**Pull request by ptgott**
Backports #28443

* Edit forScopes configurations and edit guides

Closes #26500

This change requires merging gravitational/docs#326 to add a Team scope to the docs.

This updates pages within the docs so that:

- Each page's `forScopes` configuration is accurate, especially with regard to support for Teleport Team.
- All scoped components match the `forScopes` configuration for each page. For this, I used the linter introduced by gravitational/docs#327.

* Respond to alexfornuto feedback

### CODE REVIEW

**Review by stevenGravy — APPROVED** [review]

**Comment by ptgott** [comment]
I'm going to convert this one to draft and give it another look. While backporting #28443 to v12, I noticed some issues that might have carried over to this branch as well.

File: docs/pages/includes/edition-prereqs-tabs.mdx [hunk 1]
```diff
@@ -36,7 +54,7 @@
 
 </TabItem>
 <TabItem scope={["cloud"]}
-  label="Teleport Cloud">
+  label="Teleport Enterprise Cloud">
```
Comment (alexfornuto) [inline docs/pages/includes/edition-prereqs-tabs.mdx:57]: @ptgott Should this remain just "Cloud" for instructions that apply to both Team and Enterprise Cloud, and have the commercial prereqs partial for when only Enterprise Cloud applies?

Comment (ptgott) [reply docs/pages/includes/edition-prereqs-tabs.mdx:57]: When I created the original PR, I assumed that Team wouldn't require the Enterprise `tctl` client, so I made a separate tab. If Team does require Enterprise `tctl`, though, I think we can use "Teleport Team/Enterprise Cloud" as the label and use the same tab item for both Team and Enterprise Cloud.

Comment (alexfornuto) [reply docs/pages/includes/edition-prereqs-tabs.mdx:57]: I've confirmed that teams users should use enterprise `tctl`.

Comment (ptgott) [reply docs/pages/includes/edition-prereqs-tabs.mdx:58]: I've backported https://github.com/gravitational/teleport/pull/29024 into this PR

**Review by alexfornuto — APPROVED** [review]

**Pull request merged**